src/components/scs/ScSheet/calculations/helperFunctions.js

```diff
@@ -0,0 +1,55 @@
+export const reduceWeights = arr => {
+  const value = arr.reduce((acc, obj) => {
+    return acc + obj.weight;
+  }, 0);
+  return value;
+};
+
+export const multiplyWeightByScoreArr = arr => {
+  const mapArr = arr.map(obj => obj.weight * obj.evaluation);
+  const reducedArr = mapArr.reduce((acc, item) => acc + item);
+  return reducedArr;
+};
+
+export const round = (value, precision) => {
+  const multiplier = Math.pow(10, precision || 0);
+  return Math.round(value * multiplier) / multiplier;
+};
+
+export const determineFinalPercentage = score => {
+  let value;
```

Comment:
I would suggest online solution (but only as inspiration, I think that s the matter of 'taste'):
```
export const determineFinalPercentage = score => {
if (score <= 1.5) return 75;
if (score < 2 && score >= 1.5) return 100;
if (score < 2.5 && score >= 2) return 125;
if (score < 3 && score >= 2.5) return 150;
if (score < 3.5 && score >= 3) return 175;
if (score < 4 && score > 3.5) return 200;
if (score < 4.5 && score > 4) return 350;
if (score >= 4.5) return 300;
return 100;
```

Reply:
Yeah, you're right - it looks cleaner!